Towards renaming ALF to CPC, alfc to ethos (#11082)
cvc5/cvc5: docs/proofs/output_cpc.rst

> Proof format: Cooperating Proof Calculus

Reviewer: It looks like most of the text here is > 80 char per line.

Author: Addressed

Proof format: Cooperating Proof Calculus
========================================

> Using the flag :ref:`proof-format-mode=cpc <lbl-option-proof-format-mode>`, cvc5

Reviewer: ```suggestion
Using option :ref:`proof-format-mode=cpc <lbl-option-proof-format-mode>`, cvc5
```

outputs proofs in the Cooperating Proof Calculus proof format.

This calculus was designed to faithfully represent cvc5's internal reasoning.
As a disclaimer, this means that it treats certain operators differently from
the SMT-LIB standard.
As an example, cvc5 uses mixed arithmetic internally, where integers and reals
can appear together.
A comprehensive list of these differences can be found in the Eunoia definition
of CPC, as described below.

> An efficient C++ proof checker (Ethos) is available
> `here <https://github.com/cvc5/ethos>`_ which can check proofs in the CPC
> format.

Reviewer: ```suggestion
`Ethos <https://github.com/cvc5/ethos>`_, an efficient proof checker written in C++
allows to check proofs in the CPC format.
```

For a quick start, the cvc5 repository contains a

> :cvc5repo:`script <contrib/get-ethos-checker>` which will download and install

Reviewer: ```suggestion
:cvc5repo:`script <contrib/get-ethos-checker>` to download and install
```
